.travis.yml

```diff
@@ -5,7 +5,8 @@ services:
 - docker
 cache:
   directories:
-    - $HOME/.m2
+    - "$HOME/.m2"
+    - "$HOME/.npm"
```

Comment:
For my information, what is stored here? I thought npm does not use a global path, but just messes around with the node_modules dir

Reply:
From the documentation page
````
# keep the npm cache around to speed up installs
cache:
  directories:
  - "$HOME/.npm"
````
https://docs.npmjs.com/cli/ci.html

In a fresh container:
````
cd $HOME/.npm/
~/.npm # du -sh
43.6M	.
````
The actual content can be found here:
https://gist.github.com/iperdomo/042af3dafe282012e3cb66230a51da2b

`npm ci` deletes the local `node_modules` if exists, before installing the versions used in `package-lock.json`